client/my-sites/site-settings/site-owner-transfer/confirmation-transfer.tsx

```diff
@@ -36,6 +48,20 @@ export function ConfirmationTransfer( {
 		confirmTransfer( confirmationHash );
 	}, [ confirmTransfer, confirmationHash ] );
 
+	if ( isEmailSent ) {
+		return (
+			<Notice status="is-success" showDismiss={ false }>
+				<div data-testid="email-sent">
```

Comment:
If we are not adding a related test, let's remove `data-testid="email-sent"`

Reply:
Makes sense! I have removed the div itself.